packages/noports_core/lib/src/sshnp/util/sshnpd_channel/sshnpd_default_channel.dart

```diff
@@ -66,8 +68,16 @@ mixin SshnpdDefaultPayloadHandler on SshnpdChannel {
       }
 
       logger.info('Verified signature of msg from ${params.sshnpdAtSign}');
-      logger.info('Setting ephemeralPrivateKey');
+
       ephemeralPrivateKey = daemonResponse['ephemeralPrivateKey'];
+      logger.info('Received ephemeralPrivateKey: $ephemeralPrivateKey');
+
+      sessionAESKeyString = daemonResponse['sessionAESKey'];
+      logger.info('Received sessionAESKey: $sessionAESKeyString');
+
+      sessionIVString = daemonResponse['sessionIV'];
+      logger.info('Received sessionIV: $sessionIVString');
+
```

Comment:
Should we be logging these values?

Reply:
Could pose a risk to someone who is screen sharing, or sharing logs for debugging...

Reply:
Now logging the encrypted values